webaverse/app
# Add game settings manager

## Describe your changes
Adds a game settings manager, which controlls such game settings as canFly and canNarutoRun. Could also hold things like 2D world info in the future.

## What are the steps for a QA tester to test this pull request?
1. Go to main scene and fly or naruto run -- should work
2. Go to gamesettings.scn and try the same -- should be blocked

Builds on https://github.com/webaverse/app/pull/3421
